cmd/tests/cmd_run_test.go

```diff
@@ -1865,6 +1865,50 @@ func TestUIRenderOutput(t *testing.T) {
 	}
 }
 
+func TestUIRenderWebDashboard(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		flag      string
```

Comment:
```suggestion
```
It shouldn't work anymore. Or am I missing something?

Reply:
You are right, sorry. Fixed now.